csharp/src/Ice/Property.cs

```diff
@@ -17,6 +18,12 @@ public string
         return _pattern;
     }
 
+    public string
+    defaultValue()
```

Comment:
This should be a C# property.

Reply:
I'll clean up this class in a follow-up pr.